example.rb

```diff
@@ -75,32 +80,32 @@ def run_sample
         file.write("Hello, World!")
         file.close()
    
-        puts "Temp file = " + full_path_to_file
-        puts "\nUploading to Blob storage as blob" + local_file_name
+        puts "\nCreated a temp file: " + full_path_to_file
+        puts "\nUploading to Blob storage as blob: " + local_file_name
 
         # Upload the created file using local_file_name for the blob name
-        blob_service.create_block_blob(container.name, local_file_name, full_path_to_file)
+        blob_client.create_block_blob(container.name, local_file_name, full_path_to_file)
 
         # List the blobs in the container
-        puts "\n List blobs in the container"
-        blobs = blob_service.list_blobs(container_name)
+        puts "\nList blobs in the container"
+        blobs = blob_client.list_blobs(container_name)
         blobs.each do |blob|
-            puts "\t Blob name #{blob.name}"   
+            puts "\tBlob name #{blob.name}"   
         end  
         
         # Download the blob(s).
         # Add '_DOWNLOADED' as prefix to '.txt' so you can see both files in Documents.
         full_path_to_file2 = File.join(local_path, local_file_name.gsub('.txt', '_DOWNLOADED.txt'))
         
-        puts "\n Downloading blob to " + full_path_to_file2
-        blob, content = blob_service.get_blob(container_name,local_file_name)
+        puts "\nDownloading blob to " + full_path_to_file2
+        blob, content = blob_client.get_blob(container_name,local_file_name)
         File.open(full_path_to_file2,"wb") {|f| f.write(content)}
 
         puts "Sample finished running. Hit <any key>, to delete resources created by the sample and exit the application"
         readline()
 
         # Clean up resources. This includes the container and the temp files
-        blob_service.delete_container(container_name)
+        blob_client.delete_container(container_name)
```

Comment:
I would put this in `ensure` to make sure even there are some exception thrown, this gets executed.